plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js

```diff
@@ -1015,4 +1034,29 @@ jQuery( function ( $ ) {
 			delay: 200,
 			keepAlive: true,
 		} );
+
```

Comment:
A comment here to describe what is being done in this section of code (similar to the rest of the file) would be good (I'm not going to suggest moving your code to a separate function because that ship has already sailed in the rest of this file... wow!).